api/src/test/java/org/openmrs/OpenmrsTestsTest.java

```diff
@@ -61,12 +62,11 @@ public void shouldStartWithShould() {
 				
 				// make sure every "test" method (determined by having 
 				// the @Test annotation) starts with "testShould"
-				if (method.getAnnotation(Test.class) != null) {
+				if (method.getAnnotation(Test.class) != null && method.getAnnotation(org.junit.Test.class) != null) {
```

Comment:
```suggestion
				if (method.getAnnotation(Test.class) != null || method.getAnnotation(org.junit.Test.class) != null) {
```

Reply:
Tests will never have Junit 4 and Junit 5 annotations. Its always a good idea to make a test fail once you assume its good. `shouldStartWithShould` is supposed to fail if any method annotated with a Test annotation does not start with `should` or have `_should` in its name. So you can just open any test and delete a `should` and `_should` and see if that test here fails. Since you made. a change you should test that in a Junit 4 and 5 test method. That would have told you that with the `&&` its not checking any test method.